### ISSUE TITLE
Improving the scrolling in the Reader

### ISSUE THREAD

**Issue by Pazu5512** [Guerra24/LRReader#8]
First of all, I want to thank you for providing LRReader, it improved the usability of LANraragi, at least for me, as it behaves really quickly.

As an echo to Asinin3's suggestions in ticket #6 , I would add a little thing that will improve reading : 
- Adding a "scroll to go next page" feature
Much like what you already did with the spacebar (once it hits bottom and you press it again, it goes to the next page), is it possible to implement something similar with the mouse scroll? Most standard readers already do this : you scroll to the bottom of the page, and once you hit the page's end, when you continue scrolling, it goes to the next page. That way you can solely rely on the to read, instead of relying on the scroll + clicking on next/previous to go to the next or previous page.

Edit : Maybe apply the same philosophy for "Arrow Down", once you hit the bottom of the page, another press on the Down Arrow on the Keyboard jumps to the next page. That way it's unified between Spacebar, Down Arrow and Scroll Down.

Also, I'm not sure if it's worth opening another ticket for : 
- Make it so that when you click on a tag, it automatically opens a new tab and does a search for all the files having this same tag (not necessarily in a new tab, doing so in the current tab could also be fine, or maybe by making it open in a new tab by middle clicking the tag with the mouse, much like middle clicking an URL in Firefox/Chrome opens it in a new tab)
But I feel like it's something you already have in mind, since the tags currently seem clickable, though clicking on them doesn't do anything (or maybe something is wrong on my end)

That's all that comes to mind for now, thanks for reading.

**Comment by Guerra24** [comment]
> Adding a "scroll to go next page" feature

I see, gonna look into that. Thanks for the suggestion! 👍 

> Tag opens a new tab and does a search 

That is correct, I do have planned support for clicking on tags directly but first I need to change the archives list because right only one list can be opened, once that is done it should be very straightforward to add.
On the other hand opening in the current tab is not exactly supported due to how they are implemented, I might look into that later.
